### PR TITLE
Pedro's content

### PR DESCRIPTION
**Pull request by pataruco**
As requested, I create a folder with my details

### CODE REVIEW

File: pedro-martin-2/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Full name
+
+Pedro Alejandro Martín Valera
```
Comment (TrimHall) [inline pedro-martin-2/readme.md:3]: I didn't know your middle name is Alejandro

File: pedro-martin-2/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Favourite movie
+
+Star Wars
```
Comment (TrimHall) [inline pedro-martin-2/readme.md:7]: Please change this to Die Hard

File: pedro-martin-2/readme.md (hunk 1 above)
Comment (pataruco) [reply pedro-martin-2/readme.md]: Done

File: pedro-martin-2/readme.md (hunk 2 above)
Comment (pataruco) [reply pedro-martin-2/readme.md]: I disagree, is a good Xmas movie

**Pull request merged**